2_observations.yml

```diff
@@ -28,10 +36,31 @@ targets:
     
   out_data/reservoir_releases.csv:
     command: file.copy(from = '../delaware-model-prep/2_observations/out/reservoir_releases.csv', to = target_name)
-  
+
   out_data/reservoir_releases_lordville.csv:
     command: filter_reservoirs(out_file = target_name, in_dat = 'out_data/reservoir_releases.csv', keep = I(c('Pepacton', 'Cannonsville')))
     
   # get GRAND IDs of reservoirs above Lordville
   res_lordville:
-    command: get_grands(in_file = 'out_data/reservoir_releases_lordville.csv')
\ No newline at end of file
+    command: get_grands(in_file = 'out_data/reservoir_releases_lordville.csv')
+
+  out_data/res_io_obs.feather:
```

Comment:
Are these ins/outs a combo of model predictions and true observations? Thinking about where these should go if they are a mix. I think here is okay even if they contain some model predictions.

Reply:
Nevermind -- I see the SNTEMP predictions below in predictions. Nice!